### PR TITLE
Fixed the year to 2022 and websites.

### LINKED ISSUE #95: It's GSSoC 2022 not 2020
### What happened?

In the ReadMe, you have mentioned that it's GSSoC 2020 but we're in 2022.

Dude, are you stuck in a time warp or something?

### How can we reproduce this bug?

See the readme.

### Desktop Information (Optional)

_No response_

### Urgency (Optional)

High priority

### PR DESCRIPTION
**Pull request by setco-website**
## Issue that this pull request solves

Fixes #95.

## Proposed changes

* Fixed all the instances of 2020 to 2022.
* Fixed the old website to correct URL.

## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

## Screenshots

image

## Other information

This is a part of Demo! 😝

### CODE REVIEW

File: ReadME.md [hunk 1]
```diff
@@ -106,13 +106,13 @@ As this is a React JS application, you need the following to support your develo
...
 ## Motivation
 
-This project is part of the [GirlScript Summer of Code 2020](https://www.gssoc.tech/) Open Source Program.
+This project is part of the [GirlScript Summer of Code 2022](https://gssoc.girlscript.tech/) Open Source Program.
 
-### Frequently Asked Questions(FAQs) for [GSSoC'20](https://www.gssoc.tech/)
+### Frequently Asked Questions(FAQs) for [GSSoC '22](https://gssoc.girlscript.tech/)
 
-> This project is accepting contributions under [GirlScript Summer of Code 2020](https://www.gssoc.tech/) - three months long Open Source program conducted by [GirlScript Foundation](https://www.gssoc.tech/).
+> This project is accepting contributions under [GirlScript Summer of Code 2020](https://gssoc.girlscript.tech/) - three months long Open Source program conducted by [GirlScript Foundation](https://gssoc.girlscript.tech/).
```
Comment (praveenscience) [inline ReadME.md:113]: Dude, you forgot.

**Review by praveenscience — CHANGES_REQUESTED** [review]

File: ReadME.md (hunk 1 above)
Comment (setco-website) [reply ReadME.md:113]: Ah, I will fix it.

**Review by praveenscience — APPROVED** [review]
LGTM 👍

**Comment by setco-website** [comment]
Thanks for accepting. Please add labels.

**Comment by setco-website** [comment]
Thank you!!! 😁

**Pull request merged**